pyk/src/pyk/ktool/kprove.py

```diff
@@ -274,7 +274,7 @@ def parse_modules(
                 temp_dir=self.use_directory,
                 dry_run=True,
                 type_inference_mode=type_inference_mode,
-                args=['--emit-json-spec', ntf.name],
+                args=['--emit-json-spec', ntf.name, '--allow-new-rules'],
```

Comment:
Consider adding an integration test.

Reply:
I added a rule to `looping-spec.k`, so that `TestEmitJsonSpec` exercises adding new rules via `parse_modules`.